### PR TITLE
Add information about sharedKey auth

### PR DESCRIPTION
**Pull request by pbek**
Follow up to https://github.com/studo-app/campus-qr/commit/e95ababc38c513bf4d856b175fdead420f00e2e6#r43034310

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ The [Studo app](https://studo.com) integrates the provided Android and iOS refer
 * **External identity management**: SSO, LDAP (using JNDI) and OAuth2 are supported by Ktor authentication packages. 
 * **Logging**: Optional User-Agent and IP address logging for auditing. (Disabled by default - privacy by default)
 * **API Access**: A *shared key* can be optionally defined to grant admin access to external services.
+  * You need to pass an additional header with your request: `X-Authorization: [sharedKey]`
```
Comment (vonox7) [inline README.md:35]:
```suggestion
* **API Access**: A *shared key* can be optionally defined to grant admin access to external services: Add the `X-Authorization: [authSharedSecret]` header to the request to authenticate as administrator (sharedSecretUser).
```

**Review by vonox7 — COMMENTED** [review]
Unify wording

**Comment by pbek** [comment]
@vonox7 can you please also add information about how to set a `authSharedSecret`.

**Review by vonox7 — APPROVED** [review]

**Comment by vonox7** [comment]
@pbek as with all configuration flags in MongoDB in the configuration collection. See the *App Configuration* menu item.

**Configuration flags**

All configuration flags that are in the MongoDB database listed in collection "configuration" can be changed by the administrator. Those flags are persisted in the database even across application restarts and updates of this application. Examples for configuration flags are the imprint URL, the base URL for scanning the QR codes or the number of days after the check-in data is automatically deleted. Please contact the Studo team for further assistance or if you need more configuration parameters.

**Comment by pbek** [comment]
Thank you!

**Pull request merged**